### ISSUE TITLE
Fail to access files under linked directory

### ISSUE THREAD

**Issue by choice17** [billziss-gh/sshfs-win#442]
Host OS: Window 11 Enterprise
Target OS: 
LSB Version:    :core-4.1-amd64:core-4.1-noarch
Distributor ID: RedHatEnterprise
Description:    Red Hat Enterprise Linux release 8.4 (Ootpa)
Release:        8.4

Software: 
WinFsp 2023 Release
SSHFS-Win 2021 Release

I am able to connect target system using 
`net use \\sshfs\<username>\<pc_name>`

However, I cannot access files under linked directory created by `ln -sfn <origin_directory> <linked directory>`.
PS. I can access file directory on `<origin_directory>` directly.

I appreciate your time and effort to review this bug. Thanks.

**Comment by duapple** [comment]
Yes. I meet this sence. Accessing linux soft link is very useful. WinSCP can access soft link successfully. Please support for this function which access soft link.
